app/src/main/java/com/eduard/caloriecounter/presentation/fragments/CollectionFragment.java

```diff
@@ -0,0 +1,108 @@
+package com.eduard.caloriecounter.presentation.fragments;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.eduard.caloriecounter.R;
+import com.eduard.caloriecounter.presentation.activity.MainActivity;
+import com.eduard.caloriecounter.presentation.base.BaseFragment;
+import com.eduard.caloriecounter.presentation.model.User;
+import com.eduard.caloriecounter.presentation.presenter.CollectionContract;
+import com.eduard.caloriecounter.presentation.presenter.CollectionPresenter;
+import com.eduard.caloriecounter.presentation.utils.Calculator;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.ActionBar;
+import androidx.fragment.app.DialogFragment;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+
+public class CollectionFragment extends BaseFragment implements CollectionContract.View {
+
+//    @Inject
+//    DialogInfoFragment alertDialogInfo;
+//    @Inject
+//    CollectionPresenter collectionPresenter;
+
+    private TextView tvInfo;
+    private Button btnGenerate;
+
+    private CollectionContract.Presenter mPresenter;
```

Comment:
Whats it? Remove? it should not be here, Also why you are using "m" in the name of this variable and not using it for other variables?

Reply:
ready